runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -40,6 +40,7 @@ namespace base_loader
 
 template <typename LoaderDomain, typename SpecificLoader> class BaseLoader
 {
+protected:
```

Comment:
@wateret I turned this as protected because the alias it didn't inherited.
When it's private, `using BuiltinOperator = typename LoaderDomain::BuiltinOperator` doesn't applied on tflite_loader and circle_loader's BuiltinOperator.

Reply:
For me it is OK.